ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -872,19 +872,22 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal, TwoLineTitl
         updateShadow(for: navigationItem)
     }
 
-    private func updateFakeCenterTitleConstraints() {
-        titleViewConstraint?.isActive = false
+    private func updateTitleViewConstraints() {
+        if var titleViewConstraints = titleViewConstraints {
+            NSLayoutConstraint.deactivate(titleViewConstraints)
+            titleViewConstraints.removeAll()
```

Comment:
nit: You probably don't need to call `.removeAll()` on `titleViewConstraints` if you're going to set it to a specific array below. You could probably also avoid the last `if let` if you work on the local variable instead of the optional one. So something like
```
let viewConstraints = [topConstraint, bottomConstraint]
if usesLeadingTitle { append }
viewConstraints.activate
titleViewConstraints = viewConstraints
```